docs/cunumeric/source/api/math.rst

```diff
@@ -107,6 +107,7 @@ Miscellaneous
 .. autosummary::
    :toctree: generated/
 
+   cholesky
```

Comment:
if we remove `cholesky` from the top-level module, we don't need this change either.

Reply:
https://github.com/nv-legate/cunumeric/pull/209/commits/7dce6ac8b53d4cde689e36b9d912e253410bd821